### DISCUSSION TITLE
(plugin/music) An error occured with Apple Music

### DISCUSSION THREAD

**Discussion by RSickenberg** [lowlighter/metrics#963] [Q&A] [answered]
Hello, according to my metrics (https://raw.githubusercontent.com/RSickenberg/rsickenberg/main/github-metrics.svg) an error occurred when generating my Apple Music playlist. https://github.com/RSickenberg/rsickenberg/actions/runs/2021736900

**Comment by lowlighter** [comment, marked as answer]
It has been patched yesterday by #955

Apple completely redesigned their embed player breaking the puppeteer automation

**Reply by RSickenberg** [reply]
Oh ok, thanks!
